starters/react-typescript/server.ts

```diff
@@ -10,35 +14,39 @@ const server = Fastify({
   }
 })
 
-// @ts-ignore
+// @ts-ignore - pnpm module resolution issue
```

Comment:
I've seen this module resolution issue in normal `npm`, not `pnpm`. It happens when there's a mismatch in the Fastify version between what's inside fastify/vite and what's inside this local package.json's node_modules directory. Have you tried consolidating to make sure those versions are the same?

Reply:
I will say that this is an annoying issue that I also encounter in my non-pnpm workspace monorepo.